src/components/Footer.tsx

```diff
@@ -0,0 +1,50 @@
+// import React from 'react';
+// import classNames from 'classnames';
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)